src/asmcnc/skavaUI/screen_home.py

```diff
@@ -363,23 +362,29 @@ def on_pre_enter(self):
                 self.l.get_str('Load a file') + '...' + '[/color]'
                 )
             self.job_filename = ''
-  
-            self.job_box.range_x[0] = 0
-            self.job_box.range_x[1] = 0
-            self.job_box.range_y[0] = 0
-            self.job_box.range_y[1] = 0
-            self.job_box.range_z[0] = 0
-            self.job_box.range_z[1] = 0
-            
-            # Hack to clear any previous job files
-            try:            
-                self.gcode_preview_widget.draw_file_in_xy_plane([])
-                self.gcode_preview_widget.get_non_modal_gcode([])
-            except:
-                print 'No G-code loaded.'
 
-            self.gcode_summary_widget.hide_summary()
```

Comment:
This stuff still looks like it's not where it was before, and I don't think job recovery is dependent on it? 

Might be worth using master as a reference for how any previews should be :)